docs/apis-tools/java-client/job-worker.md

```diff
@@ -41,3 +41,55 @@ By backing off, the job worker helps Zeebe by reducing the load.
 :::note
 Zeebe's [backpressure mechanism](../../../self-managed/zeebe-deployment/operations/backpressure) can also be configured.
 :::
+
+## Job streaming
+
+:::warning
+Job streaming is an experimental feature which is still under development. It is an opt-in feature which is disabled by default.
+:::
+
+Job workers are designed to regularly poll and activate jobs. It's also possible to use them in a streaming fashion, such that jobs are automatically activated and pushed downstream to workers without requiring an extra round of polling. This greatly cuts down on overall activation latency by completely obviating the poll request.
```

Comment:
Is obviating a common term across engineering? If not, I think we could substitute this for something more familiar, like "removing".